CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through currently [open pull requests](https://github.com/ansible-collections/community.mysql/pulls).
+You can help by reviewing them. There can be good pull requests which are not merged only because there are a lack of reviews. Also there can be valuable, reviewed, but abandoned ones, which you could politely ask the original authors to complete yourself.
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+
+## Step-by-step guide how to get into development quickly
+
+We assume you use Linux as a work environment (you can use a virtual machine as well).
+
+1. If possible, make sure that you have installed and started `docker`. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements and tests are running properly isolated and in the exact same environments as in CI. You often can also use `podman` with the `docker` executable shim, so if you have that you probably do not need to install `docker`.
+
+2. Clone the [ansible-core](https://github.com/ansible/ansible) repository:
+```bash
+git clone https://github.com/ansible/ansible.git
+```
+
+Instead of installing ansible-core from source, you can also work with an already existing installation of Ansible, ansible-base or ansible-core. Simply skip steps 2 and 3 in that case.
+
+3. Go to the cloned repository and prepare the environment:
+```bash
+cd ansible && source hacking/env-setup
+pip install -r requirements.txt
+cd ~
+```
+4. Create the following directories in your home directory:
+```bash
+mkdir -p ~/ansible_collections/community/mysql
+```
+
+5. Fork the `community.mysql` repository via the GitHub web interface.
+
+6. Clone the forked repository from your profile to the created path:
+```bash
+git clone https://github.com/YOURACC/community.mysql.git ~/ansible_collections/community/mysql
+```
+
+If you prefer to use the SSH protocol:
+```bash
+git clone git@github.com:YOURACC/community.mysql.git ~/ansible_collections/community/mysql
+```
+
+
+7. Go there:
+```bash
+cd ~/ansible_collections/community/mysql
+```
+
+8. Be sure you are in the main branch:
+```bash
+git status
+```
+
+9. Show remotes. There should be the `origin` repository only:
+```bash
+git remote -v
+```
+
+10. Add the `upstream` repository:
+```bash
+git remote add upstream https://github.com/ansible-collections/community.mysql.git
+```
+
+11. Update your local `main` branch:
+```bash
+git fetch upstream
+git rebase upstream/main
+```
+
+12. Create a branch for your changes:
+```bash
+git branch -b name_of_my_branch
```

Comment:
```suggestion
git checkout -b name_of_my_branch
```

Reply:
it was `branch`??? strange... thanks:)))